app/controllers/admin/custom_contents_controller.rb

```diff
@@ -0,0 +1,13 @@
+class Admin::CustomContentsController < Admin::ApplicationController
```

Comment:
__[rubocop]__
* Style/ClassAndModuleChildren: Use nested module/class definitions instead of compact style.

<!-- Comment by [SideCI](https://sideci.com) -->